packages/local_auth/ios/Classes/LocalAuthPlugin.m

```diff
@@ -62,6 +64,25 @@ - (void)alertMessage:(NSString *)message
                                                                                    completion:nil];
 }
 
+- (void)getBiometryType:(FlutterResult)result {
+    LAContext *context = [[LAContext alloc] init];
+    NSError *authError = nil;
+    if ([context canEvaluatePolicy:LAPolicyDeviceOwnerAuthenticationWithBiometrics error:&authError]) {
+        if (authError == nil) {
+            if (@available(iOS 11.0.1, *)) {
+                if (context.biometryType == LABiometryTypeFaceID) {
+                    result(@"faceID");
+                } else if (context.biometryType == LABiometryTypeTouchID) {
+                    result(@"touchID");
```

Comment:
Let's not use platform specific constants.

If you want to show platform specific messaging you can do the following in your app:

```dart
if (Platform.isIOS && getAvailableBiometrics.contains(FINGERPRINT)) {
   // this is Touch ID.
}
```

Reply:
If I get error.code = LAErrorTouchIDNotEnrolled, the differentiation between touchID and faceID is not yet available (not yet enrolled). 

In this case, should I return ["undefined"] to detect that in dart that it would be possible (canCheckBiometrics=true) but not yet which type (getAvailableBiometrics=['undefined'])?